src/sentry/api/endpoints/group_events.py

```diff
@@ -38,63 +43,81 @@ def get(self, request, group):
         :pparam string issue_id: the ID of the issue to retrieve.
         :auth: required
         """
-
-        def respond(queryset):
-            return self.paginate(
-                request=request,
-                queryset=queryset,
-                order_by='-datetime',
-                on_results=lambda x: serialize(x, request.user),
-                paginator_cls=DateTimePaginator,
-            )
-
-        events = Event.objects.filter(group_id=group.id)
+        backend = request.COOKIES.get('eventstream', 'legacy')
```

Comment:
This is one thing that we would need to change before shipping, Either moving this to an option so that we can change it globally at runtime, or just hardcoding snuba as the default